public/client.js

```diff
@@ -2418,19 +2435,20 @@ function tintGraphics(img, colorString) {
 //create a sprite from a "thing" object found in data
 function createThing(thing, id) {
 
-    var f = 1;
+  var f = 1;
 
-    if (thing.frames != null)
-        f = thing.frames;
+  if (thing.frames != null)
+    f = thing.frames;
 
-    var sw = floor(thing.spriteGraphics.width / f);
-    var sh = thing.spriteGraphics.height;
+  return thing.spriteGraphics().then((spriteGraphics) => {
```

Comment:
Same thing here, the `then`. Everything below this is not a real change I just changed the indentation by accident and didn't change it back.